### PR TITLE
Midnight theme + formatting readme 

### PR DESCRIPTION
**Pull request by karizma**
Added a new theme

- Took shydactyl as a base and then made it dark mode. 
- Same hint mode as the the theme `dark`,

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -200,7 +200,18 @@ You can bind your own shortcuts in normal mode with the `:bind` command. For exa
...
+    - default
+    - dark (authored by @furgerf)
+    - shydactyl (authored by @atrnh)
+    - greenmat (authored by @caputchinefrobles)
+    - halloween
+    - quake
+    - quakelight
+    - midnight (authored by @karizma)
+
+    Tridactyl can also load themes from disk, which would let you use one of the themes authored by @bezmi ([bezmi/base16-tridactyl](https://github.com/bezmi/base16-tridactyl)), see `:help colors` for more information.
```
Comment (bovine3dom) [inline readme.md:214]:
```suggestion
    Tridactyl can also load themes from disk or URL, which would let you use one of the themes authored by @bezmi ([bezmi/base16-tridactyl](https://github.com/bezmi/base16-tridactyl)). See `:help colors` for more information.
```

**Comment by bovine3dom** [comment]
I don't think this should be possible but somehow this PR stops me from changing themes. If I remove `src/static/themes/midnight/midnight.css` and rebuild Tridactyl it works again.

Do you see the same thing?

**Comment by Rummskartoffel** [comment]
Cannot reproduce. In case you are doing a quick build, maybe try a full one?

@karizma: Theme looks really nice.

**Comment by karizma** [comment]
Just finished building and loading, same as Rummskartoffel, works fine for me. 

> @karizma: Theme looks really nice.

Thank you.

**Comment by bovine3dom** [comment]
I did do a full build ... but I tried again and I can't reproduce it. I don't like computers :(

LGTM, thanks!

**Pull request merged**